src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -0,0 +1,223 @@
+package net.maizegenetics.phgv2.api
+
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.vcf.VCFFileReader
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.Channel
+import net.maizegenetics.phgv2.utils.AltHeaderMetaData
+import net.maizegenetics.phgv2.utils.parseALTHeader
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.util.*
+
+class HaplotypeGraph(hvcfFiles: List<String>) {
+
+    private val myLogger = LogManager.getLogger(HaplotypeGraph::class.java)
+
+    // Map<sampleName, sampleId>
+    private lateinit var sampleNameToIdMap: Map<String, Int>
+    private lateinit var sampleNames: Array<String>
+
+    // lookup sequence checksum index
```

Comment:
This comment isn't clear to me.  Is this array used to store checksums, which can be found from an index?

Reply:
I updated the comment to be this.

// lookup the sequence checksum index
    // which is the index into seqHash
    // lookup[refRangeId][sampleId]